cvmfs/catalog.h

```diff
@@ -247,6 +248,15 @@ class Catalog : public SingleCopy {
   void FixTransitionPoint(const shash::Md5 &md5path,
                           DirectoryEntry *dirent) const;
 
+  enum ExternalDataStatus {
```

Comment:
Please add some documentation on the meaning of the constants.  The constant names should start with a `k`, like `kUnknown`.